public/app/features/provisioning/RepositoryStatusPage.tsx

```diff
@@ -23,21 +23,19 @@ import {
 import { Page } from 'app/core/components/Page/Page';
 import { useQueryParams } from 'app/core/hooks/useQueryParams';
 
+import { isNotFoundError } from '../alerting/unified/api/util';
+
 import { useGetRepositoryStatusQuery, useListJobQuery, useGetRepositoryFilesQuery, Repository } from './api';
 import { FileDetails } from './api/types';
 import { PROVISIONING_URL } from './constants';
 
 enum TabSelection {
   Files = 'files',
   Jobs = 'jobs',
-  Folder = 'folder', // the configured folder
-  Settings = 'settings',
```

Comment:
given that patrics mockup has a differnet UX for create vs update... should we stick the existing value inside the settings tab?

Reply:
Haven't seen the different mockups, but in any case why it make it less easily accessible (via a manage page tab) as compared to the direct edit link? 

Also forgot to add a comment about the folder tab - looks like its intention was to duplicated the UI from Grafana folder, which seemed redundant.